web/views/api/package_view.ex

```diff
@@ -24,11 +24,22 @@ defmodule HexWeb.API.PackageView do
         release
         |> Map.take(params)
         |> Map.put(:url, api_release_url(Endpoint, :show, package, version))
+        |> if_value(Map.has_key?(release, :retirement), &load_retirement(&1, release.retirement))
```

Comment:
it looks like we need this `if_value` because api package search currently does not return `retirement` key.

@ericmj should we return retirement in search too? according to [apiary docs](http://docs.hexpm.apiary.io/#reference/releases/packages-collection/list-all-packages) response for all packages and packages search should have the same shape? or we want to keep search payload small on purpose, in which case we should update the docs